apps/openssl.c

```diff
@@ -267,6 +271,56 @@ int main(int argc, char *argv[])
     setup_trace(getenv("OPENSSL_TRACE"));
 #endif
 
+    sec_mem_char = getenv("OPENSSL_SEC_MEM");
+    if (sec_mem_char != NULL) {
+#ifndef OPENSSL_NO_SECURE_MEMORY
+        long sec_mem = 0;
+        long sec_mem_minsize = 0;
+        char *end = NULL;
+
+        errno = 0;
+        sec_mem = strtol(sec_mem_char, &end, 10);
```

Comment:
Why force base 10 here?
Specifying memory sizes in hex seems quite reasonable to me.  Especially since page sizes are most naturally expressed in hex, octal or binary.